crates/sui-indexer-alt-framework/src/pipeline/sequential/mod.rs

```diff
@@ -119,6 +119,7 @@ pub(crate) fn pipeline<H: Handler + Send + Sync + 'static>(
         processor_tx,
         metrics.clone(),
         cancel.clone(),
+        None, // No main reader lo for sequential pipelines
```

Comment:
Does the logic for filtering out data based on `main_reader_lo` need to live on the processor (which is shared between the sequential and concurrent pipelines), or can it be pushed a bit further down the stack, e.g. into the collector or committer for concurrent pipelines, so that this code path can remain unaffected?